pkg/oci/oci.go

```diff
@@ -939,8 +939,8 @@ func ensureImage(ctx context.Context, imageStore oras.Target, image string, imgO
 		}
 	}
 
-	if len(imgOpts.PublicKeys) == 0 {
-		imgOpts.Logger.Warnf("image signature verification is disabled because no public keys were provided")
+	if !imgOpts.VerifyPublicKey {
+		imgOpts.Logger.Warnf("image signature verification is disabled due to using corresponding CLI options")
```

Comment:
I think CLI is misleading. It can also be set from the configuration file or env vairables.